misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-grssignum/src/main/java/de/metas/camel/externalsystems/grssignum/from_grs/product/processor/PushRawMaterialsProcessor.java

```diff
@@ -125,4 +126,20 @@ private static String getName(@NonNull final JsonProduct grsProduct)
 
 		return name;
 	}
+
+	@NonNull
+	private static String computeBPartnerIdentifier(@NonNull final JsonBPartnerProduct jsonBPartnerProduct)
+	{
+		if (jsonBPartnerProduct.getBPartnerMetasfreshId() != null && Check.isNotBlank(jsonBPartnerProduct.getBPartnerMetasfreshId()))
```

Comment:
```suggestion
		if (Check.isNotBlank(jsonBPartnerProduct.getBPartnerMetasfreshId()))
```

i have some IDE problems right now and can't check, but isn't "null" already covered by isNotBlank?

Reply:
yeah, it is, it but Idea still pops a warning for it, so is just for ditching that warning

Reply:
right, got it..yeeah i see